Tools/CISupport/ews-build/config.json

```diff
@@ -347,7 +347,7 @@
       "builderNames": ["Commit-Queue"]
     },
     {
-      "type": "AnyBranchScheduler", "name": "pullrequest",
+      "type": "AnyBranchScheduler", "name": "pull_request",
```

Comment:
why this change? Is this name special?

Reply:
When I first added this scheduler, I thought the name was special. It was not. But with this change, the name of a scheduler needs to match the even which is triggering it. For `merge_queue`, it would look like this:
```
{
    "type": "AnyBranchScheduler", "name": "pull_request",
    "builderNames": ["Merge-Queue"]
}
```